### ISSUE TITLE
Tizen TV emulator not started

### ISSUE THREAD

**Issue by Akash4097** [flutter-tizen/flutter-tizen#546]
After i follow all the instructions available here on this repo to install all the required tools, i try to start the tv emulator but i got the black screen with message: updating the kernel...

i am attaching a screenshot:

![tizen](https://github.com/flutter-tizen/flutter-tizen/assets/12380680/0199aeea-c30b-4acf-ab18-606ef41dd2a9)

please help me to resolve the problem. Thanks.

**Comment by swift-kim** [comment]
Are you on Windows? Does your Intel processor support VT-x and is it enabled? Actually the error itself is quite common (you can find similar issues on the Samsung Developers forum) but I have no idea what the exact cause is.

**Comment by Akash4097** [comment]
> Are you on Windows? Does your Intel processor support VT-x and is it enabled? Actually the error itself is quite common (you can find similar issues on the Samsung Developers forum) but I have no idea what the exact cause is.

yes, i am on win 10 with VT supported and enabled. can you please share any links on the forum where they exactly got this kind of error.

**Comment by JSUYA** [comment]
Does disabling cpu-vt in emulator-manager->edit->hw support cause the same problem?
I found a link related to this issue, but I don't know if it will help you.
https://forum.developer.samsung.com/t/error-to-initiate-tizen-emulator/3936/6
https://docs.tizen.org/application/tizen-studio/common-tools/emulator/

flutter-tizen team provides technologies related to tools and embedder between Tizen OS and Flutter.
The samsung developer forum may be more helpful to you.

**Comment by Akash4097** [comment]
> Does disabling cpu-vt in emulator-manager->edit->hw support cause the same problem? I found a link related to this issue, but I don't know if it will help you. https://forum.developer.samsung.com/t/error-to-initiate-tizen-emulator/3936/6 https://docs.tizen.org/application/tizen-studio/common-tools/emulator/
> 
> flutter-tizen team provides technologies related to tools and embedder between Tizen OS and Flutter. The samsung developer forum may be more helpful to you.

i tried everything mentioned in the forum you shared but nothing changed... i'll ask the help on the samsung developer forum. Thanks @JSUYA @swift-kim for your help.

**Comment by Akash4097** [comment]
> > Does disabling cpu-vt in emulator-manager->edit->hw support cause the same problem? I found a link related to this issue, but I don't know if it will help you. https://forum.developer.samsung.com/t/error-to-initiate-tizen-emulator/3936/6 https://docs.tizen.org/application/tizen-studio/common-tools/emulator/
> > flutter-tizen team provides technologies related to tools and embedder between Tizen OS and Flutter. The samsung developer forum may be more helpful to you.
> 
> i tried everything mentioned in the forum you shared but nothing changed... i'll ask the help on the samsung developer forum. Thanks @JSUYA @swift-kim for your help.

For those who face this issue will help this:

my problem got solved by turning off Memory integrity feature under memory isolation inside window security > device security > Core isolation > memory integrity.
